hs_core/templates/pages/genericresource.html

```diff
@@ -926,25 +926,22 @@ <h4 class="modal-title">Confirm files deletion</h4>
         {# ======= Resource Specific ======= #}
 
         {% if site or extended_metadata_exists or res_add_metadata or resource_edit_mode %}
-            {#  Resource specific metadata can't be edited for HIS Referenced Time Series #}
-            {% if not resource_edit_mode %}
-                {% if cm.resource_type != "CompositeResource" and cm.resource_type != "CollectionResource" %}
-                    <div class="col-xs-12 content-block">
-                        <h3>Resource Specific</h3>
-                        <div id="resourceSpecificTab">
-                            <div class="row">
-                                {% if not resource_edit_mode or cm.raccess.published %}
-                                    {% block extended_metadata %}{% endblock %}
-                                {% else %}
-                                    {% crispy metadata_form  %}
-                                {% endif %}
+            {% if cm.resource_type != "CompositeResource" and cm.resource_type != "CollectionResource" %}
+                <div class="col-xs-12 content-block">
+                    <h3>Resource Specific</h3>
+                    <div id="resourceSpecificTab">
+                        <div class="row">
+                            {% if not resource_edit_mode or cm.raccess.published %}
+                                {% block extended_metadata %}{% endblock %}
+                            {% else %}
+                                {% crispy metadata_form  %}
```

Comment:
In view mode, has the list of supported aggregation types always just been jumbled? It's a bit hard to read, but looks like this wasn't part of this regression...
![image](https://user-images.githubusercontent.com/17934193/177413340-dcbbb74c-1f1f-4ed0-b995-931099531102.png)

Reply:
sounds good https://github.com/hydroshare/hydroshare/issues/4692